core/server/services/auth/api-key/admin.js

```diff
@@ -22,27 +21,22 @@ const _extractTokenFromHeader = function extractTokenFromHeader(header) {
 };
 
 /**
- * Admin API key authentication flow:
- * 1. extract the JWT token from the `Authorization: Ghost xxxx` header
- * 2. decode the JWT to extract the api_key id from the "key id" header claim
- * 3. find a matching api_key record
- * 4. verify the JWT (matching secret, matching URL path, not expired)
- * 5. place the api_key object on `req.api_key`
- *
- * There are some specifcs of the JWT that we expect:
- * - the "Key ID" header parameter should be set to the id of the api_key used to sign the token
- *   https://tools.ietf.org/html/rfc7515#section-4.1.4
- * - the "Audience" claim should match the requested API path
- *   https://tools.ietf.org/html/rfc7519#section-4.1.3
+ * Extract JWT token from admin API URL query
+ * Eg. ${ADMIN_API_URL}/?token=${JWT}
+ * @param {string} reqUrl
  */
+const _extractTokenFromUrl = function extractTokenFromUrl(reqUrl) {
+    const {query} = url.parse(reqUrl, true);
+    return query.token;
```

Comment:
The `token` query parameter def. needs a wording check. For reference in Admin API we use a header in the following format: `Authorization: Ghost ${JWT}` and for Content API it's a `key` query parameter in format `?key=ASFWERFzxivj`. 

Think to keep it as `token`, makes sense: to be able to easier distinguish from Content API key. But I've given this just 10 seconds of thinking :sweat_smile:

Reply:
@gargol Yeah will be good to have more thoughts on this. My thinking was `token` makes sense as `token` describes perfectly what we expect as the param and still is not specific enough like `jwtToken` which we allow atm.